osu.Game/Localisation/IntroSequenceStrings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class IntroSequenceStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.IntroSequence";
+
+        /// <summary>
+        /// "Random"
+        /// </summary>
+        public static LocalisableString Random => new TranslatableString(getKey(@"random"), @"Random");
```

Comment:
Would put into `UserInterfaceStrings` as `RandomIntro` or something. This being in a single file that will never see another localisation is too much.